includes/blocks/class-sensei-ct-prev-lesson-block.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * File containing the Sensei_CT_Prev_Lesson_Block class.
+ *
+ * @package sensei
+ * @since
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_CT_Prev_Lesson_Block is responsible for rendering the '< Prev Lesson | Next Lesson >' blocks.
+ */
+class Sensei_CT_Prev_Lesson_Block {
+
+	/**
+	 * The left chevron icon.
+	 *
+	 * @var string
+	 */
+	public static $icon = '<svg viewBox="0 0 8 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+							<path d="M7 1.00002L2 6.50002L7 12" stroke="#1E1E1E" stroke-width="1.5"/>
+						   </svg>';
+
+	/**
+	 * Sensei_CT_Prev_Lesson_Block constructor.
+	 */
+	public function __construct() {
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/ct-prev-lesson',
+			[
+				'render_callback' => [ $this, 'render' ],
+			]
+		);
+	}
+
+	/**
+	 * Renders the block.
+	 *
+	 * @param array $attributes The block attributes.
+	 *
+	 * @access private
+	 *
+	 * @return string The block HTML.
+	 */
+	public function render( array $attributes = [] ) : string {
+		$lesson = get_post();
+
+		if ( empty( $lesson ) ) {
+			return '';
+		}
+
+		$urls = sensei_get_prev_next_lessons( $lesson->ID );
+
+		if ( empty( $urls['previous']['url'] ) ) {
+			return '';
+		}
+		$url = $urls['previous']['url'];
+
+		$text    = $attributes['text'] ?? __( 'Previous Lesson', 'sensei-lms' );
+		$output  = "<a class='sensei-ct-pnl-a sensei-ct-pnl-a__prev' href='$url'>";
+		$output .= self::$icon;
+		$output .= "<span class='sensei-ct-pnl-text sensei-ct-pnl-text__prev'>$text</span>";
```

Comment:
`$text` here needs to be escaped (could be just with `wp_kses_post`).

Reply:
Nice catch! Updated. 👍